src/api/handlers/channel.ts

```diff
@@ -214,27 +224,20 @@ export async function sendMessage(
   const result = await RequestManager.post(
     endpoints.CHANNEL_MESSAGES(channelID),
     {
-      ...content,
-      allowed_mentions: content.mentions
-        ? {
-          ...content.mentions,
-          replied_user: content.mentions.repliedUser,
-        }
-        : undefined,
-      message_reference: {
-        message_id: content.replyMessageID,
-      },
+      ...keysToSnake(content),
```

Comment:
No breaking changes, so could you rename `allowedMentions` option to `mentions` back?